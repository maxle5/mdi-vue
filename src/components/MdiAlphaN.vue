
<template>
  <svg :width="size" :height="size" viewBox="0 0 24 24">
      <path d="M9,7V17H11V12L13,17H15V7H13V12L11,7H9Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiAlphaN",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
  })
</script>
