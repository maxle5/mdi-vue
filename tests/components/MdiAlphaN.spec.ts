
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiAlphaN from "../../src/components/MdiAlphaN.vue";

test("MdiAlphaN snapshot", () => {
  const wrapper = mount(MdiAlphaN, {});
  expect(wrapper.html()).toMatchSnapshot();
});
