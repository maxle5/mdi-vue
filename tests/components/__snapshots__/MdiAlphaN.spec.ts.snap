// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiAlphaN snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M9,7V17H11V12L13,17H15V7H13V12L11,7H9Z" style="fill: currentColor;"></path>
</svg>"
`;
